.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         - run:
             name: Run tests
             command: |
-              poetry run pytest
+              PYTHONPATH=./dags poetry run pytest
```

Comment:
This change seems necessary at this time, but the work that @jermnelson is doing should allow us to remove the `PYTHONPATH` variable.